NuGet.config

```diff
@@ -2,6 +2,16 @@
 <configuration>
   <packageSources>
     <clear />
+    <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
+    <add key="darc-pub-dotnet-templating-0b9d7e7" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-templating-0b9d7e76/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-templating-0b9d7e7-4" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-templating-0b9d7e76-4/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-templating-0b9d7e7-3" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-templating-0b9d7e76-3/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-templating-0b9d7e7-2" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-templating-0b9d7e76-2/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-templating-0b9d7e7-1" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-templating-0b9d7e76-1/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-runtime-cf258a1-1" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-runtime-cf258a14-1/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-roslyn-analyzers-755ceda" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-roslyn-analyzers-755ceda7/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-msbuild-5e4b48a" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-msbuild-5e4b48a2/nuget/v3/index.json" />
+    <!--End: Package sources managed by Dependency Flow automation. Do not edit the sources above.-->
```

Comment:
Are these addtional sources required for an external build?  Shouldn't everything be in dotnet-public after release?

Reply:
I think you're correct, removed.